2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -3,21 +3,26 @@
 
 namespace ListsLogicAndIterations.Code
 {
-	public static class Dictionarys
-	{
-		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
-		{
-			throw new NotImplementedException();
-		}
+    public static class Dictionarys
+    {
+        public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
+        {
+            Dictionary<Guid, decimal> dict = new Dictionary<Guid,decimal>();
+            dict.Add(key, value);
+            return dict;
+        }
 
-		public static decimal GetValue(Dictionary<Guid, decimal> dict, Guid key)
-		{
-			throw new NotImplementedException();
-		}
+        public static decimal GetValue(Dictionary<Guid, decimal> dict, Guid key)
+        {
+            return dict.GetValueOrDefault(key);
+        }
 
-		public static void AddValue(Dictionary<Guid, decimal> dict, Guid key, decimal value)
-		{
-			throw new NotImplementedException();
-		}
-	}
+        public static void AddValue(Dictionary<Guid, decimal> dict, Guid key, decimal value)
+        {
+            if (dict.GetValueOrDefault(key) ==0)
```

Comment:
Though this works, dictionary class has a `ContainsKey` method that returns a bool and is preferred here.